german/Korallen/Hoehle/Meerjungfrau/meerjungfrau.md

```diff
@@ -0,0 +1,8 @@
+In der Hoehle angekommen, sehen Sie etwas Schillerndes. Im ersten Moment denken
+Sie, Sie haetten einen Schatz gefunden. Doch der vermeintliche Schatz bewegt sich 
+auf Sie zu. Es ist eine Frau! Nein, ein Fisch! Halb Frau, halb Fisch!
+
+Die Meerjungfrau schwimmt auf Sie zu und gibt Ihnen zu verstehen, dass Sie Ihnen Luft zum atmen geben
+will. Sie kommt auf Sie zu und legt Ihre Lippen sanft auf Ihre und...
+
+[Sie wachen aus dem Traum auf und besuchen das Oktoberfest](../Oktoberfest/oktoberfest.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.